src/test/CheckResultsByLevel.test.tsx

```diff
@@ -0,0 +1,323 @@
+    expect(checkHeader.at(0).props().className).toContain("coloredText");
+    checkSubheader = checkHeader.find("span");
+    expect(checkSubheader.length).toEqual(1);
+    expect(checkSubheader.at(0).text()).toEqual(" • Service Ownership check");
+    expect(checkSubheader.at(0).props().className).toContain("coloredSubtext");
+    checkContents = check.find("div.MuiAccordionDetails-root");
+    expect(checkContents.length).toEqual(1);
+    expect(checkContents.at(0).childAt(0).props().hidden).toBe(true); // ...will be enabled on
+    expect(checkContents.at(0).childAt(1).text()).toEqual("Error: This check has status pending.");
+    expect(checkContents.at(0).childAt(2).props().hidden).toBe(true);
+
+    checkCandidates = wrapper.find("div#accordion-check-3");
+    expect(checkCandidates.length).toEqual(1);
+    check = checkCandidates.at(0);
+    expect(check.props().style?.backgroundColor).toEqual("#00ff0055");
+    checkHeader = check.find("p.MuiTypography-root");
+    expect(checkHeader.length).toEqual(1);
+    expect(checkHeader.at(0).text()).toEqual("Status: passed • Service Ownership check");
+    expect(checkHeader.at(0).props().className).toContain("coloredText");
+    checkSubheader = checkHeader.find("span");
+    expect(checkSubheader.length).toEqual(1);
+    expect(checkSubheader.at(0).text()).toEqual(" • Service Ownership check");
+    expect(checkSubheader.at(0).props().className).toContain("coloredSubtext");
+    checkContents = check.find("div.MuiAccordionDetails-root");
+    expect(checkContents.length).toEqual(1);
+    expect(checkContents.at(0).childAt(0).props().hidden).toBe(true); // ...will be enabled on
+    expect(checkContents.at(0).childAt(1).text()).toEqual("Error: This check has status passed.");
+    expect(checkContents.at(0).childAt(2).props().hidden).toBe(false);
+    expect(checkContents.at(0).childAt(2).text()).toEqual("Last updated: May 11th 2023, 20:47:53 (UTC)");
+
+    checkCandidates = wrapper.find("div#accordion-check-4");
+    expect(checkCandidates.length).toEqual(1);
+    check = checkCandidates.at(0);
+    expect(check.props().style?.backgroundColor).toEqual("#00000055");
+    checkHeader = check.find("p.MuiTypography-root");
+    expect(checkHeader.length).toEqual(1);
+    expect(checkHeader.at(0).text()).toEqual("Status: upcoming_failed • Service Ownership check");
+    expect(checkHeader.at(0).props().className).toContain("coloredText");
+    checkSubheader = checkHeader.find("span");
+    expect(checkSubheader.length).toEqual(1);
+    expect(checkSubheader.at(0).text()).toEqual(" • Service Ownership check");
+    expect(checkSubheader.at(0).props().className).toContain("coloredSubtext");
+    checkContents = check.find("div.MuiAccordionDetails-root");
+    expect(checkContents.length).toEqual(1);
+    expect(checkContents.at(0).childAt(0).props().hidden).toBe(false);
+    expect(checkContents.at(0).childAt(0).childAt(0).text()).toEqual("This check will be enabled on ");
```

Comment:
As you get further into `checkContents.at(0).childAt(0).childAt(5)...` it might be better to:

1. find the element by ID
2. Assert on the whole line of text, rather than each element